satellite/rewards/partner.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package rewards
+
+import (
+	"encoding/json"
+	"os"
+	"strings"
+
+	"github.com/zeebo/errs"
+)
+
+// PartnerList defines a json struct for defining partners.
+type PartnerList struct {
+	Partners []PartnerInfo
+}
+
+// PartnerInfo contains information about a partner.
+type PartnerInfo struct {
+	Name string
+	ID   string
+}
+
+// UserAgent returns partners cano user agent.
```

Comment:
What's `cano`? a typo?

Reply:
That should be `canonical`.

Reply:
https://github.com/storj/storj/pull/3495